hashicorp/boundary-ui
# feat: 🎸 user can update clusterURL when controller unsupported

## Description

user can update clusterURL when controller unsupported



## Screenshots (if appropriate):

https://github.com/hashicorp/boundary-ui/assets/29386339/c6840d70-6cc8-45ee-a788-153a68f4e307

## How to Test

Start the boundary desktop client and connect it to a version of boundary that does not support pagination (<=0.14.3). When you start the desktop client, the `unsupported controller` modal should appear with a new button to redirect the user to the cluster url page to update it if they need to.

## Checklist:
